docs/content/en/cookbook-core.md

```diff
@@ -0,0 +1,363 @@
+---
+title: First module
+description: 'Create Core module'
+position: 13
+category: Cookbook
+---
+
+The first module will be called `Core` because it will be the main module to supervise everything
+
+> All files to be added are located in the `src` directory.
+
+<alert type="info" align="center">
+      <img src="demo/image/demo-core-module.png" alt="Core module"/>
+</alert>
+
+## Initiate file
+
+First you need to create an [initialization file](/module-config#indexjs).
+
+```javascript [@Core/index.js]
+  export default async function () {}
+```
+> The function is empty because we do not run any additional mechanisms before this module.
+
+## Configuration
+
+We create a `config` directory and in it we create a `index.js` file that contains all the configurations.
+
+
+> More information about [configuration](/module-config#main-configuration).
+
+### `name`
+Set the name of the module. The name should match the directory structure.
+```javascript
+  name: '@example/core',
+```
+
+### `order`
+Specify the order in which the module is loaded. The smaller the value, the earlier the module is loaded.
```

Comment:
the smaller value / or the smaller value is, the earlier module is loaded. I feel there is too much of "the".

Reply:
It's correct -> the value is specified, so "the"s have to stay :)